pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,78 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects", "subjectaccessreview", "localsubjectaccessreview"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "delete", "list", "patch", "update", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("users", "groups"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects", "subjectaccessreview", "localsubjectaccessreview"),
```

Comment:
list doesn't do anything for SARs and a basic user will give you list on projects, so you don't need that one either.